crypto/types/multisig/multisignature.go

```diff
@@ -35,29 +35,39 @@ func getIndex(pk crypto.PubKey, keys []crypto.PubKey) int {
 
 // AddSignature adds a signature to the multisig, at the corresponding index.
 // If the signature already exists, replace it.
-func (mSig *Multisignature) AddSignature(sig []byte, index int) {
+func AddSignature(mSig *signing.MultiSignatureData, sig signing.SignatureData, index int) {
```

Comment:
Just curious, is there a reason to move the method receiver to the first param? :)

Reply:
`MultiSignatureData` is in the `signing` package so we can't really make it a receiver...